src/models.js

```diff
@@ -698,7 +698,7 @@ module.exports = ({ cooler, isPublic }) => {
 
       return messages;
     },
-    threadReplies: async (rootId, customOptions = {}) => {
+    topicComments: async (rootId, customOptions = {}) => {
```

Comment:
Is a "thread" the same thing as a "topic"?  Maybe there's two words used to mean the same thing in various places

It might be useful to have a "vocabulary.md" document defining these words to help us use them consistently